### DISCUSSION TITLE
Why doesn't the multi-file field have a file interface?

### DISCUSSION THREAD

**Discussion by Ncturn** [directus/directus#5329] [Archive]
Hello.

I'm creating a new project with the latest version of directus and when I create a multiple files field, I realize that I only have the many-to-many interface available and when I go to create a new item I cannot upload files directly, I have to go to the files module and upload the files there.

**Comment by altezza04** [comment]
I am facing the same problem. While within the file module, it can do mutli-files upload. I don't see why not do the same for multi files field.

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/issues/5246
